docs/source/TestParameters.rst

```diff
@@ -137,6 +137,46 @@ is ``None``, which still produces an empty `AvocadoParams`_. Also, the
 `Variant`_ can also be a ``tuple(list, paths)`` or just the
 ``list`` of :class:`avocado.core.tree.TreeNode` with the params.
 
+Dumping/Loading Variants
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Depending on the number of parameters, generating the Variants can be very
+compute intensive. And since the Variants are generated as part of the Job
+execution, that compute intensive task will be executed by the systems under
+test, causing an unwanted cpu load on those systems.
+
+To avoid such situation, you can export (dump) the variants to a JSON file and
+use that file (load) on the system where the Job will be executed. Example:
```

Comment:
Do you think it'd be useful to mention one can use `jobdata/variants.json` by `--load-variants`? I think it's useful...